### ISSUE TITLE
How to get all workgroup threads to `continue` to hit a breakpoint/barrier?

### ISSUE THREAD

**Issue by hughperkins** [jrprice/Oclgrind#106]
How to get all workgroup threads to `continue` to hit a breakpoint/barrier?

I know I could do like:
```
wi 0
c
wi 1
c
wi 2
c
```
...but I have 32 threads :-P  The bug I'm investigating doesnt manifest on less than 32 threads for me, so I cant reduce the number of threads unfortunately

**Comment by jrprice** [comment]
Agree, an 'advance work-group to barrier' command would be useful.
